rapids-cmake/cython/add_rpath_entries.cmake

```diff
@@ -72,14 +72,19 @@ function(rapids_cython_add_rpath_entries)
     list(APPEND cleaned_paths "${path}")
   endforeach()
 
+  if (CMAKE_SYSTEM_NAME STREQUAL "Linux")
+    set(platform_rpath_origin "\$ORIGIN")
```

Comment:
Is this supposed to be escaped? (`$ORIGIN` vs. `\$ORIGIN`) It isn't escaped here in the original code, but it is escaped in the `install_rpath` in the other file. I'm not 100% sure how variable interpolation works in CMake so escaping may be needed.

Reply:
No need to escape $ORIGIN with CMake 3.16+